okhttp/src/main/java/okhttp3/internal/InternalKt.kt

```diff
@@ -70,3 +72,9 @@ fun Any.waitMillis(timeout: Long, nanos: Int = 0) {
 fun Any.notify() = (this as Object).notify()
 
 fun Any.notifyAll() = (this as Object).notifyAll()
+
+fun toggleMask(cursor: Buffer.UnsafeCursor, key: ByteArray) = WebSocketProtocol.toggleMask(cursor, key)
```

Comment:
What is the value in having these here as opposed to just in WebSocketProtocol? and used by reader/writer?

Reply:
They're internal in `WebSocketProtocol` and can't be consumed by the Java files. Once the reader/writer are Kotlin, we can purge the functions in `InternalKt` and things will work as intended. 

The above strategy seemed to be what we did for public types. If we're not as strict for the internal classes, then I don't mind making those functions public in `WebSocketProtocol` itself.

Reply:
Yep, I'd suggest making them public for now.  Internal is off limits anyway.

Reply:
543ddbe109061a431070a197e4c4acc290187069